content/4-about-plugins/4-1-what-is-plugin.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Plugin Mechanism
-weight: 40
+title: What is Plugin
```

Comment:
```suggestion
title: Plugins
```

Reply:
Fixed in https://github.com/coreruleset/documentation/pull/137/commits/53a0433399d443048850bf3b8de34a4eed9b087b22